modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1251,23 +1251,7 @@ public String toExampleValue(Schema schema) {
             return schema.getExample().toString();
         }
 
-        if (ModelUtils.isBooleanSchema(schema)) {
-            return "null";
-        } else if (ModelUtils.isDateSchema(schema)) {
-            return "null";
-        } else if (ModelUtils.isDateTimeSchema(schema)) {
-            return "null";
-        } else if (ModelUtils.isNumberSchema(schema)) {
-            return "null";
-        } else if (ModelUtils.isIntegerSchema(schema)) {
-            return "null";
-        } else if (ModelUtils.isStringSchema(schema)) {
-            return "null";
-        } else if (ModelUtils.isObjectSchema(schema)) {
-            return "null";
-        } else {
-            return "null";
-        }
+        return "null";
```

Comment:
The code block aims to illustrate what type to check when computing the example value.

Reply:
@wing328 I don't get your point, the previous code block returns null in all cases. Can you please elaborate more?

Reply:
My point is that for someone new to the code base, they may have no clue what type to check and what function to leverage (e.g. ModelUtils.isIneger) to check the type.

The code is put there for onboarding new users.

Reply:
Maybe it can be reverted and a note can be added that the conditionals exist as examples for new contributors?

Reply:
I would put the conditionals as comment to enlighten new contributors and just leave the code return null. I personally prefer to have cleaner code and cover the unusual case in a comment

Reply:
I often agree with this approach. However, if any of the methods are renamed, comments are often missed and this can be confusing.

From another approach, new and existing contributors might not easily reason about what a single "null" result would mean here, while the explicit cases provide clear intention.

I think commenting on either way would be acceptable. We can always change later if people have questions.